src/tracking_state.jl

```diff
@@ -51,6 +50,7 @@ function DownconvertedSignalCPU(num_ants::NumAnts{N}) where N
     )
 end
 
+@inline gen_blank_carrier_gpu(system::S, num_samples) where {CO <: CuMatrix,S <: AbstractGNSS{CO}} = StructArray{ComplexF32}((CuArray{Float32}(undef, num_samples), CuArray{Float32}(undef, num_samples)))
```

Comment:
I think this can be dismissed, if you do not save the sincos result in a vector.

Reply:
This isn't used anymore, is it? So I think you can rid of this, too.